openshift-environment-driver/src/main/java/org/jboss/pnc/environment/openshift/Configurations.java

```diff
@@ -28,30 +28,49 @@
 /**
  * @author <a href="mailto:matejonnet@gmail.com">Matej Lazar</a>
  */
-public enum Configurations {
+public class Configurations {
```

Comment:
Why did you change it from enum to class, now the implementation is more verbose and it doesn't seem to be more readable.